ddsc/core/fileuploader.py

```diff
@@ -234,7 +254,11 @@ def upload_async(data_service_auth_data, config, upload_id,
     data_service = DataServiceApi(auth, config.url)
     sender = ChunkSender(data_service, upload_id, filename, config.upload_bytes_per_chunk, index, num_chunks_to_send,
                          progress_queue)
-    return sender.send()
+    try:
+        sender.send()
```

Comment:
Some confusion here - this no longer returns the value of `sender.send()`, but the caller wasn't changed. Does it not expect a return value? The tests don't expect any return value.

Also, in the `send(self)` method (L294), it's documented that it returns a string error message on error, but I don't see any evidence of that. It only ever returns `None`